index.njk

```diff
@@ -5,32 +5,27 @@ title: Rehydrate
 
 <section class="middle">
   <div class="podcast-summary">
-    <p>A podcast where we read books together at a TV series-like pace.</p>
-    <p>Books include <i>The Three-Body Problem</i> and <i>The Long Way to a Small, Angry Planet</i></p>
+    <p>A podcast where we discuss books, music, and other media</p>
```

Comment:
Needs a period after media.

Reply:
hm, i think the period actually looks weird.. but added it